system_tests/features/steps/get_info_endpoints.py

```diff
@@ -0,0 +1,200 @@
+import nose
+import requests
+
+from config import Config
+from behave import given, when
+from flask import json
+
+
+@given('the system requests action endpoint info')
```

Comment:
The BDD/gherkin syntax suggests that the given step should set up the state https://martinfowler.com/bliki/GivenWhenThen.html, https://github.com/cucumber/cucumber/wiki/Given-When-Then.

This would mean the given step would look more like
```python
@given('the action service is running') # This is probably an empty step because the service is already running
@when('I request the action info') # This would make the http request to the endpoint
@then('I retrieve the action service info') # This would check the info is as expected
```